src/rust/ru.readmanga/src/lib.rs

```diff
@@ -0,0 +1,100 @@
+#![no_std]
+#![feature(pattern)]
+#![feature(iter_intersperse)]
+
+mod constants;
+mod helpers;
+mod parser;
+mod sorting;
+mod wrappers;
+
+use aidoku::{
+	error::Result,
+	prelude::*,
+	std::net::Request,
+	std::{String, Vec},
+	Chapter, DeepLink, Filter, Listing, Manga, MangaPageResult, Page,
+};
+
+extern crate alloc;
+
+use crate::{sorting::Sorting, wrappers::debug};
+
+#[initialize]
+pub fn initialize() {
+	// println!(
+	// 	"{}",
+	// 	Node::new_fragment(b"<a href=\"foo\"><div>foo</div></a>")
+	// 		.unwrap()
+	// 		.outer_html()
+	// 		.read()
+	// );
+	// debug!(
+	// 	"{:?}",
+	// 	WNode::new("<a href=\"foo\"><div>foo</div></a>".to_string()).attr("href")
+	// );
+}
+
+#[get_manga_list]
+pub fn get_manga_list(filters: Vec<Filter>, page: i32) -> Result<MangaPageResult> {
+	let search_url = parser::get_filter_url(&filters, &Sorting::default(), page)?;
+	let html = helpers::get_html(&search_url)?;
+	let mangas = parser::parse_search_results(&html)?;
+	let result = helpers::create_manga_page_result(mangas);
+	debug!("get_manga_list: {result:?}");
```

Comment:
We shouldn't leave in any debug logging

Reply:
Why? Does it slow code much?

Reply:
Removed

Reply:
Yeah, each call to an imported function is a little expensive, especially when it has to write and read strings in memory from the WASM into a Swift String instance. Also, it just pollutes the log.